src/hash_from_two_arrays.rs

```diff
@@ -0,0 +1,13 @@
+// http://rosettacode.org/wiki/Hash_from_two_arrays
+
+use std::collections::HashMap;
+
+fn main() {
+    let keys = ["a", "b", "c"];
+    let values = [1, 2, 3];
+
+    let mut hash = HashMap::new();
+    for (key, value) in keys.iter().zip(values.iter()) {
+        hash.insert(key, value);
+    }
```

Comment:
Ah, I definitely prefer using `collect()`, especially so we can get rid of the unnecessary `mut`. Is 

```rust
let hash = keys.iter().zip(values.iter()).collect::<HashMap<_, _>>();
```

Better or worse?